### PR TITLE
[Pathways] Final copy updates, filter option updates, and disables 2 charts

### PR DESCRIPTION
**Pull request by jovergaag**
## Description of the change

Final copy updates, filter option updates, and disables 2 charts on the supervision to prison page until the bugs are worked out.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes [#XXXX]

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally
- [ ] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/content/metric/default.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ const content: MetricCopy = {
       'The chart describes the historical supervision population over the selected "Time Period". Each data point represents the total population for the selected group on the first day of that month. For example, hovering over the "November 2020" data point on the chart will show the total number of people described in the section above on November 1, 2020. ',
   },
   prisonPopulationPersonLevel: {
-    title: "List of people incarcerated",
+    title: "List of people in prison",
```
Comment (danawillow) [inline src/core/content/metric/default.ts:40]: Based on this change it looks like we specifically want to call it "prison" in TN but "incarceration" in ND, and we don't have a specific preference for the other states. Is that correct, and if so is there somewhere this is documented so it doesn't accidentally get changed back?

File: src/core/content/page/us_tn.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ const content: StateSpecificPageCopy = {
     summary:
       "Includes people incarcerated in a TDOC prison for a felony or misdemeanor conviction. People in county jails and workhouses are not included. ",
     methodology:
-      "These charts show people incarcerated in a TDOC prison for a felony or misdemeanor conviction. People in county jails are not included.",
+      'These charts show people incarcerated in a TDOC prison for a felony or misdemeanor conviction. People in county jails are not included. The "admission reason" is determined by the original admission reason for the current sentence. As a result, movements that happened after the most recent sentence started may not be captured. For example, if someone was admitted from liberty to prison on a new court commitment for the most recent sentence, then escaped then returned during the same sentence, they will show up under the admission reason of "New court comittment."',
```
Comment (danawillow) [inline src/core/content/page/us_tn.ts:28]: typo: comittment --> commitment

File: src/core/utils/filterOptions.ts [hunk 3]
```diff
@@ -789,13 +792,13 @@ export const TnPopulationFilterOptions: PopulationFilters = {
...
-    type: FILTER_TYPES.LEGAL_STATUS,
+  [FILTER_TYPES.ADMISSION_REASON]: {
+    type: FILTER_TYPES.ADMISSION_REASON,
     title: "Admission Reason",
-    setFilters: setFilters(FILTER_TYPES.LEGAL_STATUS),
+    setFilters: setFilters(FILTER_TYPES.ADMISSION_REASON),
     options: [
       { label: "All", value: "ALL" },
-      { label: "New admission", value: "NEW_ADMISSION" },
+      { label: "New court comitment", value: "NEW_ADMISSION" },
```
Comment (danawillow) [inline src/core/utils/filterOptions.ts:801]: and here

File: src/core/content/metric/default.ts (hunk 1 above)
Comment (jovergaag) [reply src/core/content/metric/default.ts:40]: ND is the only state that calls it incarceration as far as I know, the others should all use prison.

Comment (jovergaag) [reply src/core/content/metric/default.ts:40]: It is not documented - other than in the [copy doc](https://docs.google.com/spreadsheets/d/1NaU5neS0fAsr_ljAOMi05VxXRUBxuAiEQ0ndky8vnkU/edit#gid=222114944).

**Review by danawillow — APPROVED** [review]

**Pull request merged**